js/impress.js

```diff
@@ -397,6 +397,33 @@
                 scale:     1
             };
             
+            // Hide the mouse cursor in fullscreenmode after the configured
+            // amount of milliseconds without movement
+            if(rootData.hideMouseAfter) {
+                var mouseMoveTimer = null,
+                isCursorVisible = true;
+                
+                var hideCursor = function() {
+                    //isFullscreen
+                    if(!window.screenTop && !window.screenY){
+                        mouseMoveTimer = null;
+                        document.body.style.cursor = "none";
+                        isCursorVisible = false;
+                    }
+                };
+                
+                document.onmousemove = function() {
```

Comment:
I think you should use document.addEventListener() instead. Other than that I agree with this approach.